cmd/cli/plugin/secret/registry_secret_add.go

```diff
@@ -66,6 +66,16 @@ func registrySecretAdd(cmd *cobra.Command, args []string) error {
 	}
 	registrySecretOp.Password = password
 
+	pkgClient, err := tkgpackageclient.NewTKGPackageClient(registrySecretOp.KubeConfig)
+	if err != nil {
+		return err
+	}
+
+	kc, err := kappclient.NewKappClient(registrySecretOp.KubeConfig)
+	if err != nil {
+		return err
+	}
+
 	if registrySecretOp.ExportToAllNamespaces {
 		log.Warning("Warning: By choosing --export-to-all-namespaces, given secret contents will be available to ALL users in ALL namespaces. Please ensure that included registry credentials allow only read-only access to the registry with minimal necessary scope.\n\n")
```

Comment:
Lets add `checkNamespacesList()` here as well prior to warning.

Reply:
After adding this, cyclomatic complexity increased to 16>15, so had to introduce another function issueWarningAndPromptUser to further break the parent function and resolve lint issue